hazelcast/pom.xml

```diff
@@ -190,7 +190,7 @@
         <dependency>
             <groupId>com.eclipsesource.minimal-json</groupId>
             <artifactId>minimal-json</artifactId>
-            <version>0.9.2</version>
+            <version>0.9.4</version>
```

Comment:
upgrade to the same version with oss side, otherwise compilation failures can be raised, also there was some not found classes with older version.